modules/nw-install-config-gateway-api.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * networking/about-contour-operator.adoc
+
+[id="nw-install-config-gateway-api_{context}"]
+= Installing and configuring gateway API
+
+The following guide provides instructions for using link:https://gateway-api.sigs.k8s.io/[Gateway API] with the Contour Operator on {product-title}.
+
+.Prerequisites
+
+* You installed an {product-title} cluster and the `kubectl` command line.
+* You installed the Contour Operator.
+//* Log in as a user with cluster-admin permission.
+
+.Procedure
+
+. Install the Gateway API and dependent resources:
++
+[source,terminal]
+----
+$ kubectl apply -f https://raw.githubusercontent.com/projectcontour/contour-operator/v1.13.0/examples/gateway/gateway.yaml
```

Comment:
same as https://github.com/openshift/openshift-docs/pull/33631/files#r656413438. find/replace kubectl with oc throughout.

Reply:
Thank you! Replacing.